lib/l10n/app_en.arb

```diff
@@ -0,0 +1,30 @@
+{
+  "appTitle": "Form",
+  "form": "Form",
```

Comment:
The key `form` makes no sense. You should have a name that gives you SOME sense of where this is used.